src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/model/ProductType.java

```diff
@@ -1,15 +1,22 @@
 package uk.gov.companieshouse.certifiedcopies.orders.api.model;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import static uk.gov.companieshouse.certifiedcopies.orders.api.converter.EnumValueNameConverter.convertEnumValueNameToJson;
+
 /**
  * Values of this represent the possible product types.
  */
 public enum ProductType {
-    CERTIFIED_COPY("certified-copy"),
-    CERTIFIED_COPY_SAME_DAY("certified-copy-same-day");
-
-    private String value;
-
-    private ProductType(String value) { this.value = value; }
+    CERTIFICATE,
+    CERTIFICATE_SAME_DAY,
+    CERTIFICATE_ADDITIONAL_COPY,
+    SCAN_UPON_DEMAND,
+    CERTIFIED_COPY,
+    CERTIFIED_COPY_SAME_DAY;
```

Comment:
Do we need these additional enum values if they are not used in this api? If we leave them in, every time a new product is added we would have to update each API!

Reply:
Interesting point. I was thinking that we should use the same values throughout so that the reader is not left in any doubt as to whether these are values for what is conceptually the same `ProductType` type in each of the apps that actually use it? I suppose this is another illustration of the evils of duplication.

Reply:
I should add that having all the values in there means it also reflects the API spec.

Reply:
Hmmm that's true, maybe we should ask billy, but I'll approve for now.